easylist/easylist_adservers.txt

```diff
@@ -5558,6 +5559,7 @@
 ||favourmiles.com^
 ||fb-plus.com^
 ||fb0c32d2f28c.com^
+||fb9c1069a6^
```

Comment:
Looks to be typo, you probably meant `||9f3f61c6a1.com^` which is already in EL?

Reply:
@Yuki2718 True, already fixed.